config/config/wc-config.yaml

```diff
@@ -105,41 +193,96 @@ elasticsearch:
   clientNode:
     count: 1
 
+## Configuration for fluentd
+##
```

Comment:
It would be nice to mention here a bit more about it I think. For example that this is the "system" fluentd that runs on control-plane nodes and other "reserved" nodes that the user cannot run pods on.